web/src/assets/index.ts

```diff
@@ -71,6 +72,7 @@ import PoiThumbnailPlaceholderLarge from '../../../assets/poi-thumbnail-placehol
 import PoiThumbnailPlaceholder from '../../../assets/poi-thumbnail-placeholder.jpg'
 
 export {
+  mobileIcon,
```

Comment:
🔧 Please stick to the naming conventions here, so UpperCamelCase instead of just camelCase. Also, would be nice to keep this alphabetically sorted.
```suggestion
  MobileIcon,
```